Update Project Organization (#28)
qir-alliance/.github: Project_Organization.md

### 3.1. Steering Committee

The Steering Committee is the body that is responsible for governing the

> Project, coordinating Working Groups, and managing assets. No more than two 

Reviewer: Here is my suggestion for the phrasing to add: 
"To keep discussions productive and balanced, attendance is generally limited to two attendees per organization. The Steering Committee may grant an exception to this rule for a specific meeting and purpose only."

delegates from each organization can attend a 
Steering Committee meeting.

### 3.2. Chairperson

The Project will have a Chairperson whose responsibilities include leading the
Steering Committee, organizing meetings, organizing voting, identifying new

> Working Groups, managing finance, tracking Working Group progress, ensuring

Reviewer: ```suggestion
Working Groups, managing finance, tracking Working Group progress and community forums, ensuring
```

### 3.1. Steering Committee

The Steering Committee is the body that is responsible for governing the
Project, coordinating Working Groups, and managing assets. To keep discussions 
productive and balanced, attendance is generally limited to two attendees per 
organization. The Steering Committee may grant an exception to this rule for a 
specific meeting and purpose only.

### 3.2. Chairperson

The Project will have a Chairperson whose responsibilities include leading the

> Steering Committee, organizing meetings, organizing voting, identifying new

Reviewer: ```suggestion
Steering Committee, organizing meetings, organizing voting, organizing official community forums, identifying new
```

> Working Groups, managing finance, tracking Working Group progress, ensuring

Reviewer: nit: I would put "managing finance" after "organizing voting", since it is a bit odd to have it in the middle of duties related to working groups.

Working Group meet their goals, organizing community calls and official events, 

> serving as contact and representing the QIR Alliance, and any other activities 

Reviewer: ```suggestion
serving as contact and representing the QIR Alliance, updating and maintaining the official webpage, and any other activities 
```

approved by the Steering Committee. The Chairperson may delegate responsibilities 
to another party. The Steering Committee may Approve a new Chairperson. The 
Chairperson may designate an interim chair if they may be absent from one Steering 
Committee meeting. The Chairperson can be an external person. The Chairperson doesn’t 
have to be an employee of one of the Steering Committee member organizations. 
The Chairperson doesn’t have any voting privilege unless they are also appointed 
as the designated voting delegate by one of the steering committee members. 

> The Chairperson is automatically reelected for up to 4 years. The Steering 

Reviewer: ```suggestion
The Chairperson is appointed for a duration of a year or until their affiliation changes and they can be reelected. The Steering 
```

Steering Members once a year, but the Steering Committee is free to add a new
member throughout the year.

### 3.4. Core Maintainers

Core Maintainers take an active role in maintaining content in one or more of 

> the repositories under the QIR Alliance. The (at most) two primary designated 

Reviewer: ```suggestion
the repositories under the QIR Alliance. The two primary designated 
```
